layouts/partials/collapsible-menu.html

```diff
@@ -21,38 +21,62 @@
 {{ end }}
 {{$currentSection := .Scratch.Get "currentSection" }}
 
+<!--- Sidebar layout --->
+{{ .Scratch.Set "sidebarLayout" "default" }}
+{{ if isset .Site.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Site.Params.sidebar_layout }}
+{{ end }}
+{{ if isset .Page.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Page.Params.sidebar_layout }}
+{{ end }}
+
+{{ $sidebarLayout := .Scratch.Get "sidebarLayout" }}
+
+{{ $classname := "collapsible-menu" }}
+{{ if eq $sidebarLayout "sidebar_block" }}
```

Comment:
we can replace this if block with a ternary-like call https://gohugo.io/functions/cond/

Reply:
updated, nice function!